.github/workflows/deployment.yaml

```diff
@@ -2,7 +2,9 @@ name: deployment
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
+  pull-request:
```

Comment:
Make a new file, `pull_requests.yaml` or something, and move this build + prettier check there.  We can't bundle this into deployment, since that pushes to gh-pages when it completes.

Reply:
Alright, I'll revert the commit and make the changes.